### PR TITLE
[kiel] Drop Jetifier

### LINKED ISSUE #67: [kiel] Drop Jetifier
### **FEATURE REQUEST**
Hi! Thank you for making Kiel! I've played with it for some time now and I want to use it in our project but this issue is we are trying to reduce build time by **_dropping jetifier_**.

We found out by dropping jetifier we can reduce a significant amount of build time. thank you!

### **Describe the solution you'd like**
I've forked your project and dropped jetifier, test it and it works! I just disabled the app module because of some conflicts. Anyway it's working fine in my test project. 

```
 implementation 'com.github.raquezha:kiel:1.2.1-beta'
```

### **Describe alternatives you've considered**
I hope you consider this feature request

### **Additional context**
1. Adam Bennet's Article about droping jetifier: https://adambennett.dev/2020/08/disabling-jetifier/
1. Can I Drop Jetifier Repo: https://github.com/plnice/can-i-drop-jetifier

### PR DESCRIPTION
**Pull request by raquezha**
Removed android.enableJetifier in settings.gradle also fixed import conflicts

This PR is related to #67

### CODE REVIEW

File: samples/build.gradle [hunk 1]
```diff
@@ -52,8 +52,10 @@ dependencies {
     kapt "androidx.lifecycle:lifecycle-compiler:$lifecycle_version"
 
     // Glide
-    implementation "com.github.bumptech.glide:glide:$versions.glideVersion"
-    implementation "com.github.florent37:glidepalette:$versions.glidePaletteVersion"
+    implementation ("com.github.bumptech.glide:glide:$versions.glideVersion")
+    implementation ("com.github.florent37:glidepalette:$versions.glidePaletteVersion"){
```
Comment (ibrahimyilmaz) [inline samples/build.gradle]: We may completely delete this left over dependency and its version.

**Review by ibrahimyilmaz — APPROVED** [review]

File: samples/build.gradle (hunk 1 above)
Comment (raquezha) [reply samples/build.gradle]: Sure! I didn't remove it because I thought you're using it

**Comment by ibrahimyilmaz** [comment]
Could you also add a comment about dropping Jetifier to `ChangeLog.MD` please?
e.g.:
```markdown
### Development
- Your comment
### Tahat(1.2.1)
```

**Comment by raquezha** [comment]
Sure! I'll just push it in this branch. wait

wait what's Tahat(1.2.1) ?

Update: Sorry I get it now lol

**Comment by raquezha** [comment]
@ibrahimyilmaz Done updating CHANGELOG.md

**Comment by ibrahimyilmaz** [comment]
- If we remove  this line `gw check` works very fine in pipelines.
```groovy
implementation ("com.github.florent37:glidepalette:$versions.glidePaletteVersion")
```
- It is not that important but if you remove empty line in ChangeLog in 11, this will also be very good.

We can merge this directly =)

**Comment by raquezha** [comment]
Sure I'll remove it

**Comment by ibrahimyilmaz** [comment]
We have done it! Thank you for your effort and contribution✌️.

**Comment by raquezha** [comment]
Nice! Thanks also!

**Pull request merged**